assets/schema_input.json

```diff
@@ -29,8 +34,18 @@
                         "maxLength": 0
                     }
                 ]
+            },
+            "library_layout": {
+                "type": "string",
+                "enum": ["single", "paired"],
+                "errorMessage": "library_layout should be only 'single' or 'paired' value"
+            },
+            "library_strategy": {
+                "type": "string",
+                "enum": ["metagenomic", "metatranscriptomic", "genomic", "transcriptomic", "other"],
+                "errorMessage": "library strategy should be only value from list: 'metagenomic', 'metatranscriptomic', 'genomic', 'transcriptomic', 'other'"
```

Comment:
How will the "other" case be handled from prefect? e.g. in the case metadata are wrong and we realise the data are metaG/metaT and so on, is prefect going to default everything to "other" or is a manual input needed from us? In the latter case, what does "other" cover?

Reply:
I always thought OTHER is ENA notation/option

Reply:
Ah yes, it does. I was wondering whether prefect allows for "other" values, then this is great